src/garth/stats/training_status.py

```diff
@@ -0,0 +1,158 @@
+    load_tunnel_min: int | None = None
+    load_tunnel_max: int | None = None
+    load_level_trend: str | int | None = None
+    sport: str | None = None
+    sub_sport: str | None = None
+    fitness_trend_sport: str | None = None
+    fitness_trend: int | None = None
+    training_status_feedback_phrase: str | None = None
+    training_paused: bool | None = None
+    primary_training_device: bool | None = None
+    acwr_percent: int | None = None
+    acwr_status: str | None = None
+    acwr_status_feedback: str | None = None
+    daily_training_load_acute: int | None = None
+    max_training_load_chronic: float | None = None
+    min_training_load_chronic: float | None = None
+    daily_training_load_chronic: int | None = None
+    daily_acute_chronic_workload_ratio: float | None = None
+
+    _page_size: ClassVar[int]
+    _path: ClassVar[str]
+    _data_key: ClassVar[str]
+
+    @classmethod
+    def _extract_training_data(
+        cls, response: dict, data_key: str
+    ) -> list[dict]:
+        """Extract training data from the nested API response structure."""
+        if not isinstance(response, dict):
+            return []
+
+        data_section = response.get(data_key, {})
+        if not isinstance(data_section, dict):
+            return []
+
+        payload = data_section.get("payload", {})
+        if not isinstance(payload, dict):
+            return []
+
+        # For daily endpoint, get latest training status data
+        if data_key == "mostRecentTrainingStatus":
+            latest_data = payload.get("latestTrainingStatusData", {})
+            if not isinstance(latest_data, dict):
+                return []
+            # Get the first device's data (assumes single device for now)
+            for device_data in latest_data.values():
+                if isinstance(device_data, dict):
+                    # Flatten the acuteTrainingLoadDTO data
+                    acute_load = device_data.pop("acuteTrainingLoadDTO", {})
+                    if isinstance(acute_load, dict):
+                        device_data.update(acute_load)
+                    return [device_data]
+            return []
+
+        # For weekly/monthly endpoints, get report data
+        report_data = payload.get("reportData", {})
+        if not isinstance(report_data, dict):
+            return []
+
+        # Get the first device's data (assumes single device for now)
+        for device_id, device_data in report_data.items():
+            if isinstance(device_data, list):
+                result = []
+                for entry in device_data:
+                    if isinstance(entry, dict):
+                        # Flatten the acuteTrainingLoadDTO data
+                        acute_load = entry.pop("acuteTrainingLoadDTO", {})
+                        if isinstance(acute_load, dict):
+                            entry.update(acute_load)
+                        result.append(entry)
+                return result
+
+        return []
```

Comment:
the final return statement should return the result of the happy path to make it clearer to read the code